File: foundation/foundation 3 month/lesson6/classwork/ex_3.py
k=int(input("K ni kiritign: "))
arr=dict()
for x in range(1,k+1):
	name=input(f"{x} chi maxshulot nomi: ")
	price=int(input(f"{x} chi maxsulot narxi: "))
	amount=int(input(f"{x} chi maxulotning soni: "))
	tota=price*amount
	arr[name]={'narxi':price,'miqdori':amount,'sum':tota}
print(arr)
suma=0
for x in arr:
	suma+=(arr[x]['sum'])
print(f"Jami:${suma}")
if suma>=100:
	s=suma/10
	total=suma-s
	print(f"chegirma: ${s} ")
	print(f"Siz turaydigan suma: ${total}")
	ms=tuple([suma,s,total])
	print(ms)
else:
	print(f"Sizda chegirma mavjud emas va siz tulaydigan pul: ${suma} ")
	j=([suma])
	print(j)

File: foundation/foundation 2 month/lesson2/work/ex_3.c
#include <stdio.h>
int main()
{
	int sum=0,n;
	printf("Elementlar soni: ");
	scanf("%d",&n);
	int arr[n];
	for(int i=0;i<n;i++)
	{
	scanf("%d",&arr[i]);
	sum+=arr[i];
	}
	printf("Sum: %d",sum);

}
